driver_test.go

```diff
@@ -1967,3 +1967,34 @@ func TestClientSessionKeepAliveParameter(t *testing.T) {
 	}
 	defer rows.Close()
 }
+
+func TestTimePrecision(t *testing.T) {
+	var db *sql.DB
+	var err error
+
+	if db, err = sql.Open("snowflake", dsn); err != nil {
+		t.Fatalf("failed to open db. %v, err: %v", dsn, err)
+	}
+	defer db.Close()
+
+	_, err = db.Exec("create or replace table z3 (t1 time(5))")
+	if err != nil {
+		t.Errorf("error while executing query. err : %v", err)
+	}
+	res, err := db.Query("select * from z3")
+	if err != nil {
+		t.Errorf("error while executing query. err : %v", err)
+	}
+
+	cols, _ := res.ColumnTypes()
+	pres, _, _ := cols[0].DecimalSize()
+	if pres != 5 {
+		t.Fatalf("Wrong value returned. Got %v instead of 5.", pres)
+	}
+}
+
+func init() {
+	if !flag.Parsed() {
```

Comment:
This is no longer supported in go 1.13. Please remove the whole init() function completely. See https://github.com/snowflakedb/gosnowflake/pull/267/files to look how the init should be performed.

Reply:
@ChTimTsubasa This change is no longer valid after rebase.